test/Field.test.tsx

```diff
@@ -355,3 +356,14 @@ describe('Field / FastField', () => {
     expect(getProps().field.value).toBe('Binding');
   });
 });
+
+describe('<FastField />', () => {
```

Comment:
@jaredpalmer Created its own section because this test is only for `FastField`. Also, I did not create `FastField.test.tsx` for a single test. Let me know what you think.